docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -686,6 +820,45 @@ The following table lists all the playlist update mode attributes the client can
 >
 > This Attributes support Tizen 4.0 and Higher for Mobile.
 
+## Media Controller Content Type Attributes
+
+The following table lists all the content type attributes the server can receive.
+
+**Table: Media controller content type attributes**
+
+| Attribute                        | Description                              |
+|----------------------------------|------------------------------------------|
+| **Content types**        |                                          |
+| `MC_CONTENT_TYPE_IMAGE`          | Image content type                       |
+| `MC_CONTENT_TYPE_VIDEO`          | Video content type                       |
+| `MC_CONTENT_TYPE_MUSIC`          | Music content type                       |
+| `MC_CONTENT_TYPE_OTHER`          | Other content type                       |
+| `MC_CONTENT_TYPE_UNDECIDED`      | Content type is not decided              |
+
+> **Note**
+>
+> This Attributes support Tizen 5.0 and Higher for Mobile and Wearable.
+
+## Media Controller Search Category Attributes
+
+The following table lists all the search category attributes the server can receive.
```

Comment:
Please insert a colon in the end.

The following table lists all the search category attributes the server can receive: